README.md

```diff
@@ -319,6 +329,9 @@ sqlboiler -b goose_migrations postgres
 go test ./models
 ```
 
+*Note: No `mysqldump` or `pg_dump` equivalent for Microsoft SQL Server, so generated tests must be supplemented by `tables_schema.sql` with `CREATE TABLE ...` queries*
```

Comment:
Is there no third party command line utility that can do this that we can put a dependency on?

It's also possible that after reading the table, sqlboiler itself could generate the tables_schema.sql script since it has all the db info.